Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.yml

```diff
@@ -0,0 +1,1571 @@
+category: Endpoint
+commonfields:
+  id: Carbon Black Enterprise EDR
+  version: -1
+configuration:
+- display: Server URL (e.g. https://defense.conferdeploy.net)
+  name: url
+  required: true
+  type: 0
+- display: Organization Key
+  hidden: false
+  name: organization_key
+  required: true
+  type: 0
+- display: Custom Key
+  name: custom_key
+  required: true
+  type: 4
+- display: Custom ID
+  hidden: false
+  name: custom_id
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 3 days
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3
+    months, 1 year)
+  hidden: false
+  name: fetch_time
+  required: false
+  type: 0
+- defaultvalue: '50'
+  display: Fetch limit
+  hidden: false
+  name: fetch_limit
+  required: false
+  type: 0
+description: VMware Carbon Black Enterprise EDR is an advanced threat hunting and
+  incident response solution delivering continuous visibility for top security operations
+  centers (SOCs) and incident response (IR) teams. (formerly known as ThreatHunter)
+display: Carbon Black Enterprise EDR
+name: Carbon Black Enterprise EDR
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: The ID of the alert to update. Get the ID from list_alerts command.
```

Comment:
need to change the command name to the actual one

Reply:
what do you mean?

Reply:
`list-alerts`